### PR TITLE
Allow scanning all group folders

### PR DESCRIPTION
**Pull request by PVince81**
Fixes https://github.com/nextcloud/groupfolders/issues/465

image

### CODE REVIEW

**Comment by PVince81** [comment]
now looking at my screenshot it looks like the stats are not really working

**Comment by PVince81** [comment]
fixed stats and updated screenshot in top post

File: lib/Command/Scan.php [hunk 1]
```diff
@@ -28,82 +28,111 @@
...
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use OCP\Files\Cache\IScanner;
 
 class Scan extends FolderCommand {
-	private int $foldersCounter = 0;
-	private int $filesCounter = 0;
+	private array $stats;
```
Comment (artonge) [inline lib/Command/Scan.php:36]: Not needed ?

**Review by artonge — APPROVED** [review]
Besides potential unused variable, looks good

**Review by icewind1991 — APPROVED** [review]

**Pull request merged**